### PR TITLE
Bump version from 0.8.0.dev6 -> 0.9.0.dev

### PR DESCRIPTION
**Pull request by ericl**
Following release process from https://github.com/ray-project/ray/blob/bf064dabbb9d41d73b8aed8c9d1f7ae72eef8bf9/doc/dev/RELEASE_PROCESS.rst

### CODE REVIEW

File: docker/stress_test/Dockerfile [hunk 1]
```diff
@@ -4,7 +4,7 @@ FROM ray-project/base-deps
 
 # We install ray and boto3 to enable the ray autoscaler as
 # a test runner.
-RUN pip install -U https://ray-wheels.s3-us-west-2.amazonaws.com/latest/ray-0.8.0.dev5-cp36-cp36m-manylinux1_x86_64.whl boto3
+RUN pip install -U https://ray-wheels.s3-us-west-2.amazonaws.com/latest/ray-0.9.0.dev0-cp36-cp36m-manylinux1_x86_64.whl boto3
```
Comment (edoakes) [inline docker/stress_test/Dockerfile]: Looks like this hasn't been updated in awhile. Intentional? Is this file used at all?

File: docker/tune_test/Dockerfile [hunk 2]
```diff
@@ -6,7 +6,7 @@ FROM ray-project/base-deps
 # a test runner.
 RUN conda install -y numpy
 RUN pip install -U pip
-RUN pip install -U https://ray-wheels.s3-us-west-2.amazonaws.com/latest/ray-0.8.0.dev7-cp36-cp36m-manylinux1_x86_64.whl || pip install -U https://ray-wheels.s3-us-west-2.amazonaws.com/latest/ray-0.8.0.dev6-cp36-cp36m-manylinux1_x86_64.whl
+RUN pip install -U https://ray-wheels.s3-us-west-2.amazonaws.com/latest/ray-0.9.0.dev0-cp36-cp36m-manylinux1_x86_64.whl || pip install -U https://ray-wheels.s3-us-west-2.amazonaws.com/latest/ray-0.9.0.dev0-cp36-cp36m-manylinux1_x86_64.whl
```
Comment (edoakes) [inline docker/tune_test/Dockerfile]: Second one should be `0.8.0.dev7` I think (though this is strange)

File: python/ray/experimental/sgd/examples/tf-example-sgd.yaml [hunk 3]
```diff
@@ -48,7 +48,7 @@ worker_nodes:
 
 setup_commands:
     - conda install setuptools=41.0.1=py36_0 wrapt=1.11.2 --yes # workaround to fix wrapt error
-    - ray || pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.8.0.dev3-cp36-cp36m-manylinux1_x86_64.whl
+    - ray || pip install -U https://s3-us-west-2.amazonaws.com/ray-wheels/latest/ray-0.9.0.dev0-cp36-cp36m-manylinux1_x86_64.whl
```
Comment (edoakes) [inline python/ray/experimental/sgd/examples/tf-example-sgd.yaml]: Hasn't been updated in awhile. Intentional?

**Review by edoakes — APPROVED** [review]

**Comment by pcmoritz** [comment]
So happy to get rid of the `dev` versions. Finding the right one was always a pain :)

**Comment by simon-mo** [comment]
weird the wheel is built under 0.9.0.dev0 for some reason:
https://travis-ci.com/ray-project/ray/jobs/268260522#L270

**Comment by richardliaw** [comment]
^ all of the wheel links in this PR are broken - just click the links here: https://ray.readthedocs.io/en/latest/installation.html

**Pull request merged**